### PR TITLE
test(agoric-cli): test dry run inter bid

### LINKED ISSUE #7601: cannot pass non-frozen objects error when attempting to bid on ollinet
## Describe the bug
Get a 'cannot pass non-frozen objects like...' error when attempting to bid on Ollinet

```
brendanotoole@Brendans-MacBook-Pro agoric-sdk % inter bid by-price --price 10.05 --give 15IST --maxBuy 1_000_000ATOM --from brendan   
(Error#1)
Error#1: Cannot pass non-frozen objects like {
  method: 'executeOffer',
  offer: {
    id: 'bid-1683165512314',
    invitationSpec: {
      source: 'agoricContract',
      instancePath: [Array],
      callPipe: [Array]
    },
    proposal: { give: [Object] },
    offerArgs: { maxBuy: [Object], offerPrice: [Object] }
  }
} . Use harden()

  at sendAction (packages/agoric-cli/src/lib/wallet.js:109:47)
  at placeBid (packages/agoric-cli/src/commands/inter.js:332:26)
  at Command.<anonymous> (packages/agoric-cli/src/commands/inter.js:417:15)
  at async packages/agoric-cli/src/bin-agops.js:76:3
```

## To Reproduce
1. Setup CLI for bidding (install Agoric SDK, all tools, set network env etc)
2. Attempt to bid
3. Receive below error

## Expected behavior
Bid should be successfully accepted

## Platform Environment
 - what OS are you using? what version of Node.js?

 - is there anything special/unusual about your platform?

Nope - just pulled down latest Xsnode tools and updated to MacOS to Ventura yesterday
 - what version of the Agoric-SDK are you using? (run `git describe --tags --always`)

agoric-upgrade-8-2223-g720a16fd7

### PR DESCRIPTION
**Pull request by samsiegart**
closes: https://github.com/Agoric/agoric-sdk/issues/7601

### Documentation Considerations

Added an optional `--dry-run` arg to test, updated snapshots. Non-breaking change.

### Testing Considerations

Test fails before, and passes after, https://github.com/Agoric/agoric-sdk/pull/7615

### CODE REVIEW

**Review by ivanlei — APPROVED** [review]
Thanks so much.

**Comment by samsiegart** [comment]
@Mergifyio refresh

File: packages/agoric-cli/test/test-inter-cli.js [hunk 1]
```diff
@@ -557,20 +557,11 @@ test('README: inter vbank list', async t => {
...
 test('README ex1: inter bid place by-price: printed offer is correct', async t => {
   // The README example shows "bid is broadcast" but we test only bid format.
-  const noNet = '--generate-only';
+  const noNet = '--dry-run';
   const argv =
     `node inter bid by-price --give 85IST --price 8.55 --from test-acct ${noNet}`
       .trim()
       .split(' ');
-  // TODO: txhash, height stuff
-  const expected = {
```
Comment (dckc) [inline packages/agoric-cli/test/test-inter-cli.js:566]: I was trying to keep sort of clean expected results near the front of the test and then mangle them later if necessary. oh well.

Comment (samsiegart) [reply packages/agoric-cli/test/test-inter-cli.js:566]: Gotcha, I agree the new expected value is pushing the boundaries of readability a bit. I do prefer to be more explicit in tests and minimize the amount of utilities and transforms used though.

**Pull request merged**